neurom/geom/tests/test_transform.py

```diff
@@ -28,14 +28,30 @@
 
 import neurom.geom.transform as gtr
 from neurom.core.dataformat import COLS
-from neurom.core.tree import val_iter, ipreorder
+from neurom.core.tree import val_iter, ipreorder, Tree, make_copy
+from neurom.core.neuron import Neuron, make_soma
 from neurom.ezy import load_neuron
 from nose import tools as nt
 from itertools import izip
 import numpy as np
+from copy import copy
 
+TREE = Tree(np.array([0.0, 0.0, 0.0, 1.0, 1, 1, 2]))
```

Comment:
I didn't see a good reason to limit it to numpy arrays. The data representation is an implementation detail. The only requirements on the neurite tree values are that they be indexable and sliceable, and that the elements correspond to (x, y, z, r).